### PR TITLE
allow select individual action operations

### PR DESCRIPTION
**Pull request by snopoke**
## Description
PR on top of https://github.com/dimagi/open-chat-studio/pull/811 that allows the user to select individual operations from a custom action schema.

This gives more control over which API endpoints the bot has access to without having to edit the API schema doc (which can be large)

## Demo

<div>
    <a href="https://www.loom.com/share/669ef0e93fc5487b93735dd454054ca3">
      <p>My Team | dimagi | Dimagi Chatbots - 5 November 2024 - Watch Video</p>
    </a>
    <a href="https://www.loom.com/share/669ef0e93fc5487b93735dd454054ca3">
      
    </a>
  </div>

### CODE REVIEW

**Comment by snopoke** [comment]
@coderabbit review

**Comment by snopoke** [comment]
@coderabbit resolve

File: templates/custom_actions/custom_action_form.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "generic/object_form.html" %}
+{% block form_actions %}
+  <div class="text-sm mt-2">
+    {% if object and object.operation_ids %}
```
Comment (SmittieC) [inline templates/custom_actions/custom_action_form.html:4]: I don't see `operation_ids` defined anywhere?

**Comment by SmittieC** [comment]
Just one question, otherwise it looks good to me.

File: templates/custom_actions/custom_action_form.html (hunk 1 above)
Comment (snopoke) [reply templates/custom_actions/custom_action_form.html:4]: Ah thanks. I can remove this whole template file

**Pull request merged**